library/c.lib_rev.h

```diff
@@ -2,7 +2,7 @@
 #define REVISION		4
 #define SUBREVISION		1
 
-#define DATE			"17.12.2024"
+#define DATE			"20.12.2024"
 #define VERS			"clib4.library 1.4.1"
-#define VSTRING			"clib4.library 1.4.1 (17.12.2024)\r\n"
-#define VERSTAG			"\0$VER: clib4.library 1.4.1-5d728de (17.12.2024)"
+#define VSTRING			"clib4.library 1.4.1 (20.12.2024)\r\n"
+#define VERSTAG			"\0$VER: clib4.library 1.4.1-5d728de (20.12.2024)"
```

Comment:
These changes should not be in the repo. These are changed when we do a new release. If you be so kind to revert those changes.

Reply:
I didn't make those changes. There is some automatic feature autogenerating these.